src/artstationapplication/Shape.java

```diff
@@ -153,9 +162,5 @@ void reset(){
         //pass
     }
     
-    //Temporary, remove later or find real solution
```

Comment:
It looks like your commit is overwriting a few lines down here, because I finished the picture class drop while you were still working on your feature. Can you make a new commit working from the newest version to avoid overwriting my code.